system_test/aest_perf_SUITE.erl

```diff
@@ -138,6 +136,20 @@ sync_speed(Cfg) ->
 
     [?assertEqual(A, B) || A <- AllBlocks, B <- AllBlocks, A =/= B].
 
+stay_in_sync(Cfg) ->
+    Height = ?cfg(height),
+    Nodes = [n1, n2, n3, n4, n5, n6],
+    setup_nodes(cluster(Nodes, #{
+        mine_rate => ?cfg(mine_rate),
+        source => ?cfg(source)
+    }), Cfg),
+    [start_node(N, Cfg) || N <- Nodes],
+    wait_for_startup(Nodes, Height, Cfg),
+    wait_for_value({height, Height + 500}, Nodes, ?cfg(mine_timeout) * 500, Cfg),
+
+    Blocks = [get_block(N, Height + 500, Cfg) || N <- Nodes],
+    [?assertEqual(A, B) || A <- Blocks, B <- Blocks, A =/= B].
```

Comment:
Not sure how the `A =/= B` filter plays with `?assertEqual(A, B)`. Maybe you meant to keep the node in the structure, and filter only couples of distinct nodes?

Reply:
Agree, that makes more sense. And makes the assert error better.